docs/src/users_guide.md

```diff
@@ -75,11 +75,12 @@ An example of the DSL generated for the contraction of a two qubit GHZ circuit l
 # version: 0.3.0
 # Determination of contraction plan :
 #   Method used : flow cutter
-#   Time allocated : 10
+#   Treewidth : 2
+#   Time allocated : 30
 #   Seed used : -1
 #   Returned metadata :
 #     1 : c min degree heuristic
-#     2 : c status 3 1618911763948
+#     2 : c status 3 1620299104065
```

Comment:
Out of interest, any idea what this number is?

Reply:
No not yet. I've been meaning to read the following paper to figure it out but haven't yet: https://arxiv.org/pdf/1709.08949.pdf